modules/system-info/reporters/server.php

```diff
@@ -410,4 +403,63 @@ public function get_elementor_library() {
 			'value' => 'Connected',
 		];
 	}
+
+	/**
+	 * Get permissions by keys. If empty keys, get all permissions.
+	 * Be aware that you can only use key paths that are defined in the get_paths() method.
```

Comment:
it feels odd to me...
How do I check a specific path that I need?
Maybe we should have something like this that handles regular paths:

```PHP
public function get_permissions( $path ) { ... }

// ...

$server->get_permissions( 'path/to/file.txt' );
```

And something like this that handles any "system" paths (need to work on the name) that will always return everything (the filter logic will be on the consumer side):

```PHP
public function get_system_paths_permissions() { ... }

// ...

$permissions = $server->get_system_paths_permissions();

$permissions->filter( $key === Server::KEY_PATH_WP_DIR )
```
 What do you think?

Reply:
I changed the solution, inspired by your suggestion, what do you think?